lib/components/Information/Badge/index.tsx

```diff
@@ -1 +1,33 @@
-export * from "@/ui/badge"
+import { Avatar } from "@/components/Information/Avatar"
+import { cn } from "@/lib/utils"
+import { Badge as BadgeComponent } from "@/ui/badge"
+import { ComponentProps, forwardRef } from "react"
+
+interface BadgeProps {
+  text: string
+  hasAvatar?: boolean
+  variant?: ComponentProps<typeof BadgeComponent>["variant"]
+  imageSrc?: ComponentProps<typeof Avatar>["src"]
+  alt?: ComponentProps<typeof Avatar>["alt"]
+}
+
+export const Badge = forwardRef<HTMLDivElement, BadgeProps>(
+  ({ text, hasAvatar = false, imageSrc, alt, variant = "name" }) => {
+    return (
+      <BadgeComponent
+        variant={variant}
+        className={cn(hasAvatar ? "pl-[0.15rem] pr-2" : "px-2")}
```

Comment:
I would set `flex gap-1 items-center` and prevent passing down a `className` to `Avatar` down below.

Reply:
The issue here was that the Avatar is supossed to be very close to the left, almost touching the border: 
![image](https://github.com/user-attachments/assets/8fe3037d-362a-40ee-ac29-9219834b9a25)

Because the image is rounded, then it looks fine, but if you put text so close to the border it won't look good. So for this badge with avatar I need this custom padding on the left.

I'm not really passing className to avatar, but to Badge, another would be to just make this padding part of the variant, do you want to request that change?

Reply:
Okay, I understand! I think it's ok then. The fact that we're keeping the component split between shadcn and our implementation is getting a bit in the middle. In the end, as with the rest, what matters is the public API.